static/src/javascripts/projects/common/modules/navigation/membership.js

```diff
@@ -0,0 +1,31 @@
+define(['common/modules/commercial/user-features', 'common/utils/fastdom-promise', 'Promise', 'common/utils/$'], function (userFeatures, fastdom, Promise, $) {
+    var LAST_CLASS = 'brand-bar__item--split--last';
+
+    function init() {
+        var refreshUser = new Promise(function (resolve) {
+            userFeatures.refresh();
+            resolve(userFeatures.isPayingMember());
```

Comment:
refresh does some async stuff, but not in a way that's useful here, I'm not going to call refresh, and just not show the supporter link if we don't know membership status